Shared/Samples/Densify and generalize geometry/DensifyAndGeneralizeGeometryView.SettingsView.swift

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension DensifyAndGeneralizeGeometryView {
+    struct OptionsView: View {
+        /// The view model for the sample.
+        @EnvironmentObject private var model: Model
```

Comment:
Instead of using `EnvironmentObject`, can you change to `ObservedObject`? Both works but I'm thinking of improving it. #137